lib/AppInfo/Application.php

```diff
@@ -50,6 +53,15 @@ class Application extends App implements IBootstrap {
 		// 'video/x-matroska'	// too rarely used for photos
 	];
 
+	/** Starting PHP7.4 we will be able to use spread operator (...) to merge the 2 arrays above */
+	public const MIMES = [
+		'image/png',
+		'image/jpeg',
+		'image/heic',
+		'video/mp4',
+		'video/quicktime',
```

Comment:
```suggestion
		...IMAGE_MIMES,
		...VIDEO_MIMES,
```

Reply:
I thought it was js xD

Reply:
nope but as I said PHP implemented it in 7.4, that's awesome 👍